app/components/build/AvatarWithUnknownEmailPrompt.js

```diff
@@ -94,13 +113,6 @@ class AvatarWithUnknownEmailPrompt extends React.Component {
   }
 
   getMessages(loading) {
-    // There won't be an email address if this build was created by a
```

Comment:
I believe this code here was for when we didn't request the email for the `User` type. I believe this logic was to prevent the dialog for not showing once it's been linked to a user. Since we now do the `UnregisteredUser` check, I believe we can just blow this away!